hack/report-flakes.py

```
@@ -0,0 +1,197 @@
+    "operator-grout": "grout",
+}
+
+def env_required(name):
+    val = os.environ.get(name)
+    if not val:
+        print(f"Error: {name} must be set", file=sys.stderr)
+        sys.exit(1)
+    return val
+
+def run_gh(*args):
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)